### DISCUSSION TITLE
MM+ goes black

### DISCUSSION THREAD

**Discussion by thearti121** [OnionUI/Onion#919] [Q&A]
### Checklist

- [X] I am reporting a bug in the Onion OS (Note that Onion devs can’t fix inherited issues with RA Cores)
- [X] I have reviewed the [Onion Wiki](https://github.com/OnionUI/Onion/wiki) and did not find the answer
- [X] I have searched the existing (open and closed) issues to ensure I do not log a duplicate

### Onions OS Version

4.2.0-beta-dev

### Miyoo Firmware Version

202303262339

### Provide a clear and concise description of the issue

Hi, after installing OS my console won't launch, it only goes black on OS logo, like on video I attached. 
https://streamable.com/txr3a9

### Steps to Reproduce

Before anything I updated firmware to MY354 then I formated brand new sandisk 128gb. Unziped the new 4.2.0 version I checked hiden files in windows and copied all files to card and installed OS. After rebooting console goes black on OS logo. 

### Provide any additional context or information that may help us investigate your issue

https://streamable.com/txr3a9

**Comment by Aemiii91** [comment]
Try reinstalling

**Comment by thearti121** [comment]
Did It few times but I'll try.

**Comment by Aemiii91** [comment]
Does it boot to stock when you turn it on without SD card?

**Comment by thearti121** [comment]
Yes it does

**Comment by Aemiii91** [comment]
That's a good sign at least. Did the installation run fine? No sudden exits?

**Comment by thearti121** [comment]
None, it installed without errors.

**Reply by Aemiii91** [reply]
Can you supply some screenshots of the content of your SD card? Specifically the `.tmp_update` folder.

**Reply by thearti121** [reply]
Sure
![image](https://github.com/OnionUI/Onion/assets/95141577/0d230370-f9c8-4c1b-ab67-fc8aa9ae0898)
![image](https://github.com/OnionUI/Onion/assets/95141577/aa916047-0a52-44e6-881f-1831fe4a1094)

**Reply by thearti121** [reply]
any ideas?

**Reply by Aemiii91** [reply]
We're working on a fix

**Reply by thearti121** [reply]
ok I'll wait then, thanks.

**Reply by rafaelvedovatto** [reply]
I ran into that problem with Onion 2,4-RC. Any update about the fix?

**Reply by Aemiii91** [reply]
The potential cause has been fixed. Have you tried reformatting and reinstalling?

**Reply by rafaelvedovatto** [reply]
Yep. I've just done it but the problem continues. :(

**Reply by Aemiii91** [reply]
Please create a new post describing the issue in detail. And if you can, supply a video showing the problem.

**Reply by rafaelvedovatto** [reply]
I've created post https://github.com/OnionUI/Onion/issues/1096 and will upload the video in a few.
